### PR TITLE
Remove workflow contents

### PR DESCRIPTION
**Pull request by marten**
PFE button is going away in https://github.com/zooniverse/Panoptes-Front-End/pull/5255

# Review checklist

- [ ] First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
- [ ] If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
- [ ] If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
- [ ] Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

### CODE REVIEW

File: app/controllers/api/v1/projects_controller.rb [hunk 1]
```diff
@@ -96,8 +96,7 @@ def create_workflows_export
   end
 
   def create_workflow_contents_export
-    medium = Projects::CreateWorkflowContentsExport.with(api_user: api_user, object: controlled_resource).run!(params)
-    medium_response(medium)
+    head :gone
```
[the remark this answers is not in the document]
Comment (marten) [reply app/controllers/api/v1/projects_controller.rb]: Fair point. Looks like it's an optional input to his code. It looks pretty simple to just read strings from workflows instead, I'll see if I can make a PR for it.
